src/Controller/MaintenanceConfigurationController.php

```diff
@@ -7,34 +7,41 @@
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpKernel\Attribute\AsController;
+use Symfony\Component\Routing\Attribute\Route;
 use Symfony\Contracts\Cache\CacheInterface;
 use Symfony\Contracts\Translation\TranslatorInterface;
 use Synolia\SyliusMaintenancePlugin\Exporter\MaintenanceConfigurationExporter;
 use Synolia\SyliusMaintenancePlugin\Factory\MaintenanceConfigurationFactory;
 use Synolia\SyliusMaintenancePlugin\FileManager\ConfigurationFileManager;
 use Synolia\SyliusMaintenancePlugin\Form\Type\MaintenanceConfigurationType;
 
+#[AsController]
 final class MaintenanceConfigurationController extends AbstractController
 {
     public function __construct(
-        private TranslatorInterface $translator,
-        private MaintenanceConfigurationExporter $maintenanceExporter,
-        private MaintenanceConfigurationFactory $configurationFactory,
-        private CacheInterface $synoliaMaintenanceCache,
+        private readonly TranslatorInterface $translator,
+        private readonly MaintenanceConfigurationExporter $maintenanceExporter,
+        private readonly MaintenanceConfigurationFactory $configurationFactory,
+        private readonly CacheInterface $synoliaMaintenanceCache,
     ) {
     }
 
+    #[Route('/maintenance/configuration', name: 'sylius_admin_maintenance_configuration', defaults: ['_sylius' => ['permission' => true, 'section' => 'admin', 'alias' => 'plugin_synolia_maintenance']], methods: ['GET|POST'])]
```

Comment:
can `methods` be written 
```
methods: ['GET', 'POST']
```
instead ?

Reply:
got `method not allowed` if methods are written this way :face_with_diagonal_mouth: